Formula/q/qwen-code.rb

```diff
@@ -1,10 +1,15 @@
 class QwenCode < Formula
   desc "AI-powered command-line workflow tool for developers"
   homepage "https://github.com/QwenLM/qwen-code"
-  url "https://registry.npmjs.org/@qwen-code/qwen-code/-/qwen-code-0.0.11.tgz"
-  sha256 "88a561e5882793c52ec6e6946eb736b418501cacbeceb47120ce25f87db22894"
+  url "https://registry.npmjs.org/@qwen-code/qwen-code/-/qwen-code-0.0.12.tgz"
+  sha256 "439fdd35bc2f7e0cc2cc7a20f927a8184b98319e57049128f3fe4b8629c6b792"
   license "Apache-2.0"
 
+  livecheck do
+    url :homepage
+    regex(/(\d+(?:\.\d+)+)/i)
+  end
+
```

Comment:
```suggestion
```

Reply:
sounds good, but right now it still tracks the nightly release